cmd/minishift/cmd/start.go

```diff
@@ -172,6 +179,9 @@ func runStart(cmd *cobra.Command, args []string) {
 	hostVm := startHost(libMachineClient)
 	registrationUtil.RegisterHost(libMachineClient)
 
+	// Forcibly set nameservers when configured
+	minishiftNetwork.AddNameserversToInstance(hostVm.Driver, getSlice(configCmd.NameServers.Name))
```

Comment:
This will add nameserver to any OS minishift is being run. Is this intended?

Reply:
Intended, as this can be a general issue for startup of a VM. it is an issue when LCOW had been enabled, but also can occur on xhyve: https://github.com/kubernetes/minikube/issues/1151#issuecomment-296328921